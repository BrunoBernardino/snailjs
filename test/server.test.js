import request from 'supertest';
import server from 'index.js';

test('/server', async () => {
  const response = await request(server.callback()).get('/server');
  expect(response).toBeDefined();

  expect(response.status).toEqual(200);
  expect(response.body).toMatchSnapshot();
});
