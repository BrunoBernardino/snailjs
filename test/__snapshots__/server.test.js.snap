// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/server 1`] = `Object {}`;
